tests/5.1/assume/test_assume_no_parallelism.c

```diff
@@ -0,0 +1,44 @@
+//===--- test_assume_no_parallelism.c ---------------------------------------===//
+//
+//  OpenMP API Version 5.1 Aug 2021
+//
+//  Using assume no_parallelism, the compiler can be optimized as it knows that a
+//  structured block does not contain any parallelism.
+//
+////===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+
+#define N 1024
+
+int errors, i;
+
+int test_assume_no_parallelism() {
+    #pragma omp assume no_parallelism
```

Comment:
I am trying to distinguish the cases where there is no_openmp vs. no_parallelism. How about we add a omp rt routine here and report the value ?

Something like:
int arr[N];

#pragma omp assume no_parallelism
{
         int x = omp_get_thread_num(); // should be 0
         for(i = 0; i < N; i++){
             arr[i] = i + x; 
         }
}